src/test/url-handling.test.js

```diff
@@ -330,11 +330,10 @@ describe('search strings', function() {
     ['calltree', 'stack-chart', 'flame-graph'].forEach(tabSlug => {
       dispatch(changeSelectedTab(tabSlug));
       const urlState = urlStateReducers.getUrlState(getState());
-      const { query } = urlStateToUrlObject(urlState);
-      if (!query.search) {
-        throw new Error('Could not find the search query string');
-      }
-      expect(query.search).toBe(callTreeSearchString);
+      const queryString = getQueryStringFromUrlState(urlState);
```

Comment:
I don't really understand the motivation for this change, and prefer the old implementation, since I don't have to care about encoding URI components.